.github/workflows/connector_teams_review_requirements.yml

```diff
@@ -39,4 +39,5 @@ jobs:
         with:
           status: ${{ steps.get-mandatory-reviewers.outputs.MANDATORY_REVIEWERS  }}
           token: ${{ secrets.OCTAVIA_4_ROOT_ACCESS }}
+          request-reviews: true
```

Comment:
This action now supports auto requesting reviews! Specially since the check isn't required, I think it's important to add else reviews can be missed